package.json

```diff
@@ -4,14 +4,18 @@
   "private": true,
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
-    "@gnosis.pm/safe-apps-sdk": "^0.1.1",
-    "@gnosis.pm/safe-react-components": "^0.1.3",
+    "@gnosis.pm/safe-apps-sdk": "https://github.com/gnosis/safe-apps-sdk.git#25a1afe",
```

Comment:
We should if possible use a stable (on npm released) version

Reply:
here we can, because Mikhail release it today.